src/Slider.jsx

```diff
@@ -94,6 +94,21 @@ class Slider extends React.Component {
     this.onChange({ value });
   }
 
+  onKeyboard(e) {
+    const valueMutator = utils.getKeyboardValueMutator(e);
+
+    if (valueMutator) {
+      utils.pauseEvent(e);
+      const state = this.state;
+      const oldValue = state.value;
```

Comment:
😅 no need to obey current  code style, just use what you think is better.

It's just because that we do not have enough time to refactor this component right now, and PR is welcome ~